sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -136,3 +136,11 @@ DROP TABLE IF EXISTS `pma_table_coords`;
 #IfTable pma_table_info
 DROP TABLE IF EXISTS `pma_table_info`;
 #EndIf
+
+#IfMissingColumn x12_partners x12_submitter_id
+ALTER TABLE `x12_partners` ADD COLUMN `x12_submitter_id` tinyint(1) DEFAULT NULL AFTER `x12_dtp03`;
```

Comment:
rec removing the `AFTER` clause to avoid weird things happening (since really just a cosmetic thing to order the table columns)